src/global.css

```diff
@@ -0,0 +1,4 @@
+body {
+  margin: 0px;
+  padding: 0px;
+}
```

Comment:
Nice! Can you also add this too, please?
```
* { box-sizing: border-box; }
```

Reply:
actually we might not need , we can use https://mui.com/material-ui/react-css-baseline/

let me check it out